src/main/java/edu/illinois/library/cantaloupe/async/ThreadPool.java

```diff
@@ -31,9 +31,7 @@ private String getThreadID() {
         abstract String getThreadNamePrefix();
 
         public Thread newThread(Runnable runnable) {
-            Thread thread = new Thread(runnable);
```

Comment:
This is the new Thread(...) call inside the newThread(...) method used by the _PriorityThreadFactory classes